packages/router/src/lib/operations/auction.ts

```diff
@@ -155,6 +139,24 @@ export const newAuction = async (
     });
   }
 
+  // getting the swap rate from the receiver side config
+  const swapRate = (1 - parseFloat(config.chainConfig[receivingChainId].feePercentage.toString()) / 100).toString();
```

Comment:
Swap rate and fee are two different things. We need to do the following:
- Get swap rate from AMM (leave a placeholder for this, hardcoded to `1`).
- Add additional `0.05%` fee hardcoded.